### ISSUE TITLE
Docs: How does reitit compare to compojure-api

### ISSUE THREAD

**Issue by miikka** [metosin/reitit#289]
Inspired by [this Reddit post](https://www.reddit.com/r/Clojure/comments/bwydk2/reitit_vs_compojureapi_for_creating_apis/), we should add a section to the [FAQ](https://github.com/metosin/reitit/blob/master/doc/faq.md) about how reitit differs from compojure-api and which one is recommended by Metosin (answer: reitit).

**Comment by ikitommi** [comment]
Good point, there is already comparison to Compojure, bit outdated that too https://cljdoc.org/d/metosin/reitit/0.3.7/doc/misc/faq#how-does-reitit-differ-from-compojure
